### PR TITLE
Add get_statistics to python SDK

### PR DESCRIPTION
**Pull request by zhilingc**
Adding the implementation on the python SDK side. I was considering doing aggregation at the SDK level but nixed that idea in favour of doing it server-side, so the python get_statistics method only wraps the GetFeatureStatistics grpc call.

### CODE REVIEW

File: protos/feast/core/CoreService.proto [hunk 1]
```diff
@@ -225,7 +227,7 @@ message ListProjectsResponse {
 message GetFeatureStatisticsRequest {
     // Optional filter which filters returned statistics by selected features. These
     // features must be present in the data that is being processed.
-    repeated string feature_refs = 1;
+    repeated string feature_ids = 1;
```
Comment (woop) [inline protos/feast/core/CoreService.proto]: This should be feature refs right?

File: sdk/python/feast/client.py [hunk 2]
```diff
@@ -600,6 +605,85 @@ def get_online_features(
...
+            if start_date is not None:
+                request.start_date.CopyFrom(
+                    Timestamp(seconds=int(start_date.strftime("%s")))
+                )
+            if end_date is not None:
+                request.end_date.CopyFrom(
+                    Timestamp(seconds=int(end_date.strftime("%s")))
+                )
+
+        return self._core_service_stub.GetFeatureStatistics(
```
Comment (woop) [inline sdk/python/feast/client.py]: Might need self._connect_serving()?

File: protos/feast/core/CoreService.proto (hunk 1 above)
Comment (zhilingc) [reply protos/feast/core/CoreService.proto]: No, feature refs should refer to the FeatureRef object for consistency

**Pull request merged**